operator/bucketcontroller/pipeline.go

```diff
@@ -2,14 +2,22 @@ package bucketcontroller
 
 import (
 	"context"
-
 	"github.com/crossplane/crossplane-runtime/pkg/event"
 	"github.com/crossplane/crossplane-runtime/pkg/resource"
 	"github.com/minio/minio-go/v7"
 	cloudscalev1 "github.com/vshn/provider-cloudscale/apis/cloudscale/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	// EndpointName is the endpoint field name in credential secret
+	EndpointName = "ENDPOINT"
+	// RegionName is the region field name in credential secret
+	RegionName = "REGION"
+	// BucketName is the bucket field name in credential secret
+	BucketName = "BUCKET_NAME"
+)
+
```

Comment:
In this file we also need some `toConnectionDetails` func that returns a map with all the 5 properties. That way it can be consumed by a composition and ultimately stored in the connection secret for the user.